pkg/controller/migplan/validation.go

```diff
@@ -335,9 +346,32 @@ func (r ReconcileMigPlan) validatePodLimit(plan *migapi.MigPlan) error {
 		})
 	}
 
+	if len(nsWithNodeSelectors) > 0 {
+		msgFormat := "Found Pods with `Spec.NodeSelector` or `Spec.NodeName` set in namespaces: [%s]. " +
+			"These fields will be cleared on Pods restored into the target cluster."
+		plan.Status.SetCondition(migapi.Condition{
+			Type:     NsHaveNodeSelectors,
+			Status:   True,
+			Reason:   NodeSelectorsDetected,
+			Category: Warn,
+			Message: fmt.Sprintf(msgFormat,
+				strings.Join(nsWithNodeSelectors, ",")),
```

Comment:
We should separate the list of namespace with a comma and a space ", "

Reply:
Done!